sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -900,12 +903,12 @@ export class ApplicationService extends RecordDataModelService<Application> {
       },
       relations: {
         applicationException: true,
-        currentAssessment: { offering: true },
+        currentAssessment: { offering: true, studentAppeal: true },
```

Comment:
The appeal needs to come from preceding application's current assessment not the changed(going to be changed application's current assessment)